Common/include/option_structure.hpp

```diff
@@ -297,16 +297,17 @@ static const map<string, ENUM_MAT_COMPRESS> MatComp_Map = CCreateMap<string, ENU
  * \brief types of interpolators
  */
 enum ENUM_INTERPOLATOR {
-  NEAREST_NEIGHBOR = 0,   /*!< \brief Nearest Neigbhor interpolation */
-  ISOPARAMETRIC = 1, /*!< \brief Isoparametric interpolation */
-  CONSISTCONSERVE = 2,/*!< \brief Consistent & Conservative interpolation (S.A. Brown 1997). Utilizes Isoparametric interpolation. */
+  NEAREST_NEIGHBOR 	= 0,   	/*!< \brief Nearest Neigbhor interpolation */
+  ISOPARAMETRIC 	= 1,	/*!< \brief Isoparametric interpolation */
+  CONSISTCONSERVE 	= 2,	/*!< \brief Consistent & Conservative interpolation (S.A. Brown 1997). Utilizes Isoparametric interpolation. */
+  SLIDING_MESH		= 3, 	/*!< \brief Sliding Mesh Approach E. Rinaldi 2015 */
```

Comment:
Truth be told the method is flux conservative for a set special cases only (planar surfaces basically, and actually not even for them in general).
I would avoid the term CONSERVATIVE then, probably WEIGHTED_FLUX is the most appropriate. What do you think?